core/scripts/cli/admin.js

```diff
@@ -0,0 +1,122 @@
+const inquirer = require("inquirer");
+const abiDecoder = require("../../../common/AbiUtils.js").getAbiDecoder();
+
+async function decodeGovernorProposal(artifacts, id) {
+  const Governor = artifacts.require("Governor");
+  const governor = await Governor.deployed();
+  const proposal = await governor.getProposal(id);
+
+  console.group();
+  console.log("Retrieved Admin Proposal for ID:", id);
+  console.log("Proposal has", proposal.transactions.length, " transactions");
+  for (let i = 0; i < proposal.transactions.length; i++) {
+    console.group();
+    console.log("Transaction", i);
+
+    const transaction = proposal.transactions[i];
+
+    // Give to and value.
+    console.log("To: ", transaction.to);
+    console.log("Value (in Wei): ", transaction.value);
+
+    if (!transaction.data || transaction.data.length === 0 || transaction.data === "0x") {
```

Comment:
in which cases would `transaction.data === "0x"`?

Reply:
When I was testing this, an empty data field (i.e. just an eth transfer) has a return value of "0x".